localisation/00_wc_artifact_localisation.csv

```diff
@@ -228,3 +228,6 @@ whispers_intrigue_1;Murmurs of the Deep Below;Murmures des profondeurs;Raunen au
 whispers_intrigue_1_desc;When someone holds this book, dark whispers creep into the corner of their mind...;Quand quelqu'un tient ce livre, de sombres murmures se glissent dans le coin de son esprit...;Wenn jemand dieses Buch in der Hand h�lt, schleicht sich ein dunkles Fl�stern in die Winkel [From.GetHerHis]es Geistes...;;;;;;;;;;;x
 whispers_learning_1;Augurs of Eldritch Insights;Pr�sages indicibles;Unheimliche Einblicke;;;;;;;;;;;x
 whispers_learning_1_desc;When someone holds this book, dark whispers creep into the corner of their mind...;Quand quelqu'un tient ce livre, de sombres murmures se glissent dans le coin de son esprit...;Wenn jemand dieses Buch in der Hand h�lt, schleicht sich ein dunkles Fl�stern in die Winkel [From.GetHerHis]es Geistes...;;;;;;;;;;;x
+# Any #;;;;;;;;;;;;;;
+heart_of_yshaarj;Heart of Y'Shaarj;;;;;;;;;;;;;x
+heart_of_yshaarj_desc;A diseased, icy mass of flesh seething with Void energies, is the actual heart of the deceased Old God, Y'Shaarj. When Y'Shaarj was torn apart by the titan Aman'Thul, its monstrous heart was the largest part of the Old God that remained intact.;;;;;;;;;;;;;x
```

Comment:
Removed a comma
```suggestion
heart_of_yshaarj_desc;A diseased, icy mass of flesh seething with Void energies is the actual heart of the deceased Old God, Y'Shaarj. When Y'Shaarj was torn apart by the titan Aman'Thul, its monstrous heart was the largest part of the Old God that remained intact.;;;;;;;;;;;;;x
```

Reply:
Fixed in 739d4f273c3de62674c1defd2ce6cf1605cc8ef7.